homeworks/aleksey.poddubnii_AlekseyPoddubnii/homework_6/user.js

```diff
@@ -0,0 +1,34 @@
+
+const user = {};
+
+Object.defineProperty(user, 'name', {
+    set: (input) => {
+        this.name = input.toLowerCase().trim().replace(/\b\w/g, name => name.toUpperCase());
+    },
+    get: () => this.name,
```

Comment:
your `set` and `get` properties referencing arrow functions will effect global scope, this is a bad side effect

you can make it more reliable with function wrapper, or use different getter and setter syntax

Reply:
ohh, okay, not problem.. i think it will be with function wrapper, some example saw at MDN.